requirements/common.txt

```diff
@@ -18,7 +18,7 @@ pillow  # Required for image processing
 prometheus-fastapi-instrumentator >= 7.0.0
 tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.11, < 0.11
-llguidance>=0.6.15; platform_machine == "x86_64" or platform_machine == "arm64"
+llguidance >= 0.7.2, < 0.8.0; platform_machine == "x86_64" or platform_machine == "arm64" or platform_machine == "aarch64"
```

Comment:
is `arm64` actually valid? that may have just been a mistake ..

Reply:
it's arm64 on mac and aarch64 on linux; go figure...

Reply:
how convenient.